### DISCUSSION TITLE
Please add v0.6.0 to GitHub Releases

### DISCUSSION THREAD

**Discussion by stephenwade** [ExistentialAudio/BlackHole#778] [General]
Please publish new versions as releases in GitHub. https://github.com/ExistentialAudio/BlackHole/releases

This is how, e.g., Homebrew can learn that there is a new version available.

https://github.com/Homebrew/homebrew-cask/blob/master/Casks/b/blackhole-2ch.rb

Also, this is probably why the shields.io badge in the README is out of date:
[![Release](https://img.shields.io/github/v/release/ExistentialAudio/BlackHole)](https://github.com/ExistentialAudio/BlackHole/releases)
https://img.shields.io/github/v/release/ExistentialAudio/BlackHole

**Comment by devinroth** [comment]
Done. Homebrew update coming really soon.
